code/datums/periodic_news.dm

```diff
@@ -1,119 +1,113 @@
+			 	to a refugee from one of the Tau Ceti Research Stations, their station, in order
+			 	to increase revenue, has refactored several of their facilities to perform experiments
+			 	on live humans, including virology research, genetic manipulation, and \"feeding them
+			 	to the slimes to see what happens\". Allegedly, these test subjects were neither
+			 	humanified monkeys nor volunteers, but rather unqualified staff that were forced into
+				the experiments, and reported to have died in a \"work accident\" by Ark Soft Inc."}
+	author = "Unauthorized"
+
+/datum/news_announcement/bluespace_research/announcement
+
+	round_time = 60*20
+	message = {"The new field of research trying to explain several interesting spacetime oddities,
+				also known as \"Bluespace Research\", has reached new heights. Of the several
+				hundred space stations now orbiting in Tau Ceti, fifteen are now specially equipped
+				to experiment with and research Bluespace effects. Rumours have it some of these
+				stations even sport functional \"travel gates\" that can instantly move a whole research
+				team to an alternate reality."}
+
+/datum/news_announcement/random_junk/cheesy_honkers
+
+	author = "Assistant Editor Carl Ritz"
+	channel_name = "The Gibson Gazette"
+	message = {"Do cheesy honkers increase risk of having a miscarriage? Several health administrations
+				say so!"}
+	round_time = 60 * 15
+
+/datum/news_announcement/random_junk/net_block
+	author = "Assistant Editor Carl Ritz"
+	channel_name = "The Gibson Gazette"
+	message = {"Several corporations banding together to block access to 'wetskrell.nt', site administrators
+	claiming violation of net laws."}
+	round_time = 60 * 50
+
+/datum/news_announcement/random_junk/found_ssd
+	channel_name = "Nyx Daily"
+	author = "Doctor Eric Hanfield"
+
+	message = {"Several people have been found unconscious at their terminals. It is thought that it was due
+				to a lack of sleep or of simply migraines from staring at the screen too long. Camera footage
+				reveals that many of them were playing games instead of working and their pay has been docked
+				accordingly."}
+	round_time = 60 * 90
+
+/datum/news_announcement/lotus_tree/explosions
+
+	channel_name = "Nyx Daily"
+	author = "Reporter Leland H. Howards"
+
+	message = {"The newly-christened civillian transport Lotus Tree suffered two very large explosions near the
+				bridge today, and there are unconfirmed reports that the death toll has passed 50. The cause of
+				the explosions remain unknown, but there is speculation that it might have something to do with
+				the recent change of regulation in the Moore-Lee Corporation, a major funder of the ship, when M-L
+				announced that they were officially acknowledging inter-species marriage and providing couples
+				with marriage tax-benefits."}
+	round_time = 60 * 30
+
+datum/news_announcement/food_riots/breaking_news
+
```

Comment:
Put a leading slash here, like:
`/datum/news_announcement/food_riots/breaking_news`

And remove the blank line following it.